packages/automerge-repo-react-hooks/src/useDocument.ts

```diff
@@ -58,5 +60,5 @@ export function useDocument<T>(
     handle.change(changeFn, options)
   }
 
-  return [doc, changeDoc]
+  return [doc, changeDoc] as const
```

Comment:
This tells TypeScript to infer that the return value is a tuple and not an array, which frees us up from having to annotate the return type manually (which is good because it's easy to get the return type annotation slightly wrong, as it was in this case).